Tests/YamsTests/EmitterTests.swift

```diff
@@ -109,6 +109,20 @@ class EmitterTests: XCTestCase {
             }
         }
     }
+
+    func testSortedKeys() throws {
+        let node: Node = [
+            "key3": "value3",
+            "key2": "value2",
+            "key1": "value1"
+        ]
+        let yaml = try Yams.serialize(node: node)
+        let expected = "key3: value3\nkey2: value2\nkey1: value1\n"
+        XCTAssertEqual(yaml, expected)
```

Comment:
Will this be flaky on Linux like `testEncodingTopLevelStructuredSingleClass` is right now because the order isn't guaranteed?

Reply:
This won't be flaky on Linux, because this test does not use `Swift.Dictionary` that causing issue on Linux.